microk8s-resources/wrappers/run-containerd-with-args

```diff
@@ -2,11 +2,6 @@
 
 set -ex
 
-# Re-exec outside of apparmor confinement
-if [ -d /sys/kernel/security/apparmor ] && [ "$(cat /proc/self/attr/current)" != "unconfined" ]; then
-    exec aa-exec -p unconfined -- "$0" "$@"
-fi
-
```

Comment:
Shouldn't you remove also the line
```bash
apparmor_parser -r $SNAP/containerd-profile
```
below?

Reply:
We were asked to apply our own profile by the security team.  If this has changed, we can remove it.